### PR TITLE
Created endpoint for All_Modules_Page.

### PR DESCRIPTION
**Pull request by KristinaDudnyk**
In this PR:
+ route for  All_Modules_Page
+ replacing components on home page
+ creation of endPoint for  All_Modules_Page, which receives structured data from two tables (modules & topics)

image

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -4,32 +4,29 @@ const app = express();
 
 const cors = require("cors");
 const port = process.env.PORT || 5005;
-const pool = require("./database/db");
+const db = require("./database/db");
```
Comment (ElenaBarker) [inline server/index.js:7]: Hi Kristina,
May I ask why you changed 'pool' to 'db'? I think things were working fine before. Let's maybe keep those things and not change them, please.

File: server/endPoints-file/all_modules_pageEndPoint.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  topics.module_id,
+  topics.topic_name,
+  topics.description AS topic_description,
+  topics.reference_link,
+  topics.test_link
+  FROM topics
+  INNER JOIN modules
+  ON modules.id = topics.module_id
+  `;
+  // const query = `
```
Comment (ElenaBarker) [inline server/endPoints-file/all_modules_pageEndPoint.js:19]: Could you please delete the commented code if you are not going to use it? Thank you

**Review by ElenaBarker — COMMENTED** [review]
Kristina, as Adib mentioned in Wednesday's call, could you please add an image of your component? This way, we can all see how it looks before approval. Thank you.

File: server/index.js (hunk 1 above)
Comment (KristinaDudnyk) [reply server/index.js]: Hi Elena, thanks for your comment. My renaming wouldn't change much. But if we already have started this way, you're right, better leave it how it was

File: server/endPoints-file/all_modules_pageEndPoint.js (hunk 2 above)
Comment (KristinaDudnyk) [reply server/endPoints-file/all_modules_pageEndPoint.js:19]: this query is here for future inpruvement

**Pull request merged**